private/buf/cmd/buf/command/push/push.go

```diff
@@ -165,6 +167,12 @@ If you set the --%s flag and/or --%s flag yourself, then the value(s) will be us
 			createDefaultLabelFlagName,
 		),
 	)
+	flagSet.BoolVar(
+		&f.IgnoredUnnamed,
+		ignoreUnnamedFlagName,
+		false,
+		"Ignore unnamed modules. Unnamed modules must not be depended on by any named modules.",
```

Comment:
> Unnamed modules must not be depended on by any named modules.

Is this true? It seems that modules that are being pushed cannot depend on unnamed modules, but not the other way around?

Reply:
I think thats correct but wording is confusing. What about?

> Named modules must not have any unnamed dependencies.

Reply:
Oh yes, that is a lot less confusing to me ^^"